src/main/dumpxdr.cpp

```diff
@@ -116,6 +118,28 @@ xdr_printer(const PublicKey& pk)
     return KeyUtils::toStrKey<PublicKey>(pk);
 }
 
+std::string 
+xdr_printer(const MuxedAccount& muxedAccount)
+{
+    std::stringstream res;
+    switch (muxedAccount.type())
+    {
+        case KEY_TYPE_ED25519:
+            res << KeyUtils::toStrKey(toAccountID(muxedAccount));
+            break;
+        case KEY_TYPE_MUXED_ED25519:
+            res << "{ id = " << muxedAccount.med25519().id << ", ";
```

Comment:
I would really prefer if this printed on multiple lines, but I don't want to block this PR because of that. I have opened issue #2543 to track that.